### DISCUSSION TITLE
15.8.7

### DISCUSSION THREAD

**Discussion by davidjgraph** [jgraph/drawio-desktop#806] [General]
## Releases Notes for 15.8.7
[Windows Installer](https://github.com/jgraph/drawio-desktop/releases/download/v15.8.7/draw.io-15.8.7-windows-installer.exe)
[Windows No Installer](https://github.com/jgraph/drawio-desktop/releases/download/v15.8.7/draw.io-15.8.7-windows-no-installer.exe)
[macOS - Universal](https://github.com/jgraph/drawio-desktop/releases/download/v15.8.7/draw.io-universal-15.8.7.dmg)
Linux - [deb](https://github.com/jgraph/drawio-desktop/releases/download/v15.8.7/drawio-amd64-15.8.7.deb), [snap](https://snapcraft.io/drawio), [AppImage](https://github.com/jgraph/drawio-desktop/releases/download/v15.8.7/drawio-x86_64-15.8.7.AppImage) or [rpm](https://github.com/jgraph/drawio-desktop/releases/download/v15.8.7/drawio-x86_64-15.8.7.rpm)
[Google Chrome OS](https://chrome.google.com/webstore/detail/drawio-desktop/pebppomjfocnoigkeepgbmcifnnlndla/reviews)

Windows intel x32 releases are marked -ia32-

**ChangeLog:**

- Fixes event problems in Atlas theme and language selection icon. #798  
- Updates to [draw.io core 15.8.7](https://github.com/jgraph/drawio/blob/v15.8.7/ChangeLog). All changes from 15.8.5 to 15.8.7, inclusive, are added in this build.

<hr /><em>This discussion was created from the release <a href='https://github.com/jgraph/drawio-desktop/releases/tag/v15.8.7'>15.8.7</a>.</em>

**Comment by HenryVN** [comment]
It's very useful for everyone use MacOS. Thank you so much
